### DISCUSSION TITLE
Missing keywords in autocomplete menu

### DISCUSSION THREAD

**Discussion by Lun4m** [rust-lang/rust-analyzer#17142] [Q&A] [answered]
Hei, I'm using rust-analyzer in nvim with [nvim-cmp](https://github.com/hrsh7th/nvim-cmp) as the completion engine.
The problem I'm having is that the `async` and `move` keywords don't appear in the autocomplete menu, while others like `enum` or `impl` come up just fine. So I'm not sure if the problem is in rust-analyzer, in nvim-cmp, or somewhere else. 
Does anyone have any idea?

**Comment by Veykril** [comment, marked as answer]
Those are not implemented as completions yet https://github.com/rust-lang/rust-analyzer/blob/56bee2ddafa6177b19c631eedc88d43366553223/crates/ide-completion/src/completions/expr.rs#L1-L326

**Reply by Lun4m** [reply]
Thanks, I'll take a peek! :)
